recipes/eefolium/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
Since the license is now included in the PyPI tarball you can remove this file.

Reply:
I have removed the LICENSE file. Thanks.